src/shared-endpoints.ts

```diff
@@ -0,0 +1,135 @@
+//type Constructor = new (...args: any[]) => {};
+import { GenericAPIResponse } from './util/requestUtils';
+import RequestWrapper from './util/requestWrapper';
+
+export default class SharedEndpoints {
+    protected requestWrapper: RequestWrapper; // XXX Is there a way to say that Base has to provide this?
+    
+    //------------Market Data Endpoints------------>
+    
+    getOrderBook(params: {
+        symbol: string;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/orderBook/L2', params);
+    }
+    
+    /**
+    * @deprecated use getTickers() instead
+    */
+    getLatestInformation(params?: {
+        symbol?: string;
+    }): GenericAPIResponse {
+        return this.getTickers(params);
+    }
+
+    getTickers(params?: {
+        symbol?: string;
+    }): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/tickers', params);
+    }
+    
+    getSymbols(): GenericAPIResponse {
+        return this.requestWrapper.get('v2/public/symbols');
+    }
+    
+    /**
+    * @deprecated use getLiquidations() instead
+    */
+    getPublicLiquidations(params: {
+        symbol: string;
+        from?: number;
+        limit?: number;
+        start_time?: number;
+        end_time?: number;
+    }): GenericAPIResponse {
+        return this.getLiquidations(params);
+    }
```

Comment:
Same as `getLatestInformation`:
- getLiquidations can stay in the shared endpoints
- getPublicLiquidations (deprecated from inverse) can stay in inverse